pontoon/base/static/js/l20n.js

```diff
@@ -0,0 +1,2831 @@
+var L20n = (function () {
+  'use strict';
+
+  /*eslint no-magic-numbers: [0]*/
+
+  const locales2rules = {
```

Comment:
l20n.js is an external library, sorry if that wasn't clear. We should probably move these kinds of script into a `lib`subfolder.